### ISSUE TITLE
srid option causes constant schema update

### ISSUE THREAD

**Issue by HenningCash** [jsor/doctrine-postgis#30]
We are using the doctrine postgis extension with Symfony 3.3. If we are running `bin/console doctrine:schema:update --dump-sql|--force` there is always one query that requires execution:

```sql
SELECT UpdateGeometrySRID('m_geo', 'geom', 3857);
```

The schema definition for this field is:
```xml
<field name="geom" type="geometry" >
    <options>
        <option name="srid">3857</option>
    </options>
</field>
```

This happens as soon you define the srid option. I guess this happens because the schema compare tool reads the option value from database as integer and the srid from the XML definition as string.

**Comment by jsor** [comment]
Thanks for the report. Could give the current master a try? This should be fixed in f1c0d131c0f3ca8715c648ba4eeae6c8e3013d45.

**Comment by HenningCash** [comment]
yes, thank you – works like a charm! 👍

**Comment by jsor** [comment]
Fix released in [v1.4.1](https://github.com/jsor/doctrine-postgis/releases/tag/v1.4.1).
